### PR TITLE
Multiple auth headers in NC12

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Ref: #1034

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -179,7 +179,7 @@ dependencies {
     compile name: 'touch-image-view'
     compile 'com.android.support:multidex:1.0.1'
 
-    compile 'com.github.nextcloud:android-library:1.0.18'
+    compile 'com.github.nextcloud:android-library:authInNC12-SNAPSHOT'
```
Comment (tobiasKaminsky) [inline build.gradle]: Needs to changed to new library version after testing!

File: build.gradle [hunk 2]
```diff
@@ -179,7 +179,7 @@ dependencies {
     compile name: 'touch-image-view'
     compile 'com.android.support:multidex:1.0.1'
 
-    compile 'com.github.nextcloud:android-library:authInNC12-SNAPSHOT'
+    compile 'com.github.nextcloud:android-library:1.0.19'
```
Comment (AndyScherzinger) [inline build.gradle]: should be 1.0.20 :)

**Comment by AndyScherzinger** [comment]
👍

**Comment by mario** [comment]
👍

**Pull request merged**